src/components/Footer.js

```diff
@@ -14,6 +14,9 @@ const Footer = () => (
       </a>
       .
     </p>
+    <p className="text-white">
```

Comment:
I think we should do something to differentiate it as a link, either a slightly different colour or underline or something, WDYT?